Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -899,6 +900,8 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didComp
 
     if (auto networkDataTask = [self existingTask:task])
         networkDataTask->didCompleteWithError(error, networkDataTask->networkLoadMetrics());
+    else if (auto* webSocketTask = [self existingWebSocketTask:task])
+        webSocketTask->didClose(WebCore::ThreadableWebSocketChannel::CloseEventCodeAbnormalClosure, emptyString());
```

Comment:
On the surface, it seemed weird to me that we might tell the existing web socket task there's an error (`CloseEventCodeAbnormalClosure`) even if there's no `NSError` passed in here. (e.g. it's a "normal" completion from the NSURLSession standpoint)

In discussion with Alex, he mentioned it's an error for a web socket to end without having sent a close frame, and that's what's captured here.

I'd like a comment here to that effect to make that crystal clear to future code spelunkers.